test/jdk/javax/naming/spi/providers/InitialContextTest.java

```diff
@@ -280,38 +298,18 @@ private static Result java(Map<String, String> properties,
         String cp = classpath.stream()
                 .map(Path::toString)
                 .collect(Collectors.joining(File.pathSeparator));
+//        System.setProperty("test.noclasspath", "true");
```

Comment:
This newly introduced commented code can be removed.

Reply:
Sorry, I missed this. Removed in current commit.